docs/source/3x/client_quant.md

```diff
@@ -42,9 +37,4 @@ python main.py
 > [!TIP]
 > For Linux systems, users need to configure the environment variables appropriately to achieve optimal performance. For example, set the `OMP_NUM_THREADS` explicitly. For processors with hybrid architecture (including both P-cores and E-cores), it is recommended to bind tasks to all P-cores using `taskset`.
 
-### Optimal Performance and Peak Memory Usage
-
-Below are approximate performance and memory usage figures conducted on a client machine with 24 cores and 32GB of RAM. These figures provide a rough estimate for quick reference and may vary based on specific hardware and configurations.
-
-- 7B models (e.g., [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf)): the quantization process takes about 65 seconds, with a peak memory usage of around 6GB.
-- 1.5B models (e.g., [Qwen/Qwen2-1.5B-Instruct](https://huggingface.co/Qwen/Qwen2-1.5B-Instruct)),  the quantization process takes about 20 seconds, with a peak memory usage of around 5GB.
+When working with 7B models (e.g., [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf)), `RTN` quantization is a quick process, finishing in tens of seconds and using several GB of RAM. However, for the highest accuracy, `GPTQ` algorithm is recommended, but be prepared for a longer quantization time.
```

Comment:
Suggest change order and minor update:  " RTN quantization is a quick process, finishing in tens of seconds and using several GB of RAM when working with 7B models, e.g., [meta-llama/Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf). However, for the higher accuracy, `GPTQ` algorithm is recommended, but be prepared for a longer quantization time.

Reply:
Thanks, updated.